index.bs

```diff
@@ -3288,27 +3404,67 @@ To <dfn>get the base network event data</dfn> given |request| and |redirect coun
      settings|' [=environment settings object/global object=], let |context id|
      be the [=browsing context id=] for that context.
 
+1. Let |redirect count| be |request|'s [=redirect count=].
+
 1. Let |timestamp| be a [=time value=] representing the current date and time in UTC.
 
+1. If |intercepts| is not [=list/empty=], let |is blocked| be true, otherwise
+   let |is blocked| be false.
+
 1. Let |params| be [=/map=] matching the <code>network.BaseParameters</code>
    production, with the <code>request</code> field set to |request data|, the
    |navigation| field set to <code>navigation</code>, the <code>context</code>
    field set to |context id|, the <code>timestamp</code> field set to
-   |timestamp|, and the <code>redirectCount</code> field set to |redirect count|.
+   |timestamp|, the <code>redirectCount</code> field set to |redirect count|, the
+   <code>isBlocked</code> field set to |is blocked|, and <code>intercepts</code>
+   field set to |intercepts| if |is blocked| is true, or omitted otherwise.
+
+1. Return |params|
+
+</div>
 
-1. Return (|related browsing contexts|, |params|)
+#### The network.Body Type #### {#type-network-Body}
+
+<pre class="cddl local-cddl remote-cddl">
+network.Body = network.StringBody / network.Base64Body;
+
+network.StringBody = {
+  type: "string",
+  value: text,
+}
+
+network.Base64Body = {
+  type: "base64",
+  value: text
+}
+</pre>
+
+The <code>network.Body</code> type represents a request or response body,
```

Comment:
@sadym-chromium @OrKoN @foolip FYI:

On CDP, there seems to be no type for "requests", only for "responses".

The relevant _response_ types are

[Network.getResponseBody](https://vanilla.aslushnikov.com/?Network.getResponseBody) or [Network.getResponseBodyForInterception](https://vanilla.aslushnikov.com/?Network.getResponseBodyForInterception) or [Fetch.getResponseBody](https://vanilla.aslushnikov.com/?Fetch.getResponseBody) or [Fetch.getResponseBodyForInterception](https://vanilla.aslushnikov.com/?Fetch.getResponseBodyForInterception).

What should we do for "requests"?

Reply:
Resolving. I misunderstood this.